src/cobald/daemon/runners/trio_runner.py

```diff
@@ -56,9 +60,7 @@ def _run_trio_blocking(self):
 
     async def _manage_payloads_trio(self):
         self._trio_token = trio.lowlevel.current_trio_token()
-        # buffer of 256 is somewhat arbitrary but should be large enough to rarely stall
-        # and small enough to smooth out explosive backlog.
-        self._submit_tasks, receive_tasks = trio.open_memory_channel(256)
+        self._submit_tasks, receive_tasks = trio.open_memory_channel(float("inf"))
```

Comment:
Are there any consequences to have an `inf` memory channel?

Reply:
It means we don't smooth things out if someone decides to spawn many, many tasks. I don't think that ever happens unless something else is broken to begin with.

I've added a new comment to clarify *why* this should be `inf` anyway (so that we cannot deadlock the event loop).